modules/dev-server/consul_1_14_compat.tf

```diff
@@ -3,6 +3,6 @@
 
 locals {
   // parse the consul version from the provided image: ["1", "12", "2"]
-  consul_image_version_parts = regex(":.*(\\d+)[.](\\d+)[.](\\d+)", var.consul_image)
-  is_consul_1_14_plus        = tonumber(local.consul_image_version_parts[1]) >= 14
+  # consul_image_version_parts = regex(":.*(\\d+)[.](\\d+)[.](\\d+)", var.consul_image)
```

Comment:
Temporarily commented out this to accept images of the format `hashicorpreview/consul-enterprise:1.17-dev` (without a patch version). This will be fixed when the RC for consul-enterprise gets released